packages/bra/vtex.country-data-settings/configuration.json

```diff
@@ -168,10 +168,17 @@
     "minimal": [
       [
         {
-          "name": "postalCode"
+          "name": "street"
```

Comment:
won't this break the layout in cases where the user enters the postal code but the API doesn't autofill the street?

Reply:
suree, it was added by mistake. ty